edns.go

```diff
@@ -148,6 +148,16 @@ func (rr *OPT) SetDo(do ...bool) {
 	}
 }
 
+// Z returns the Z part of the OPT RR as a uint16 with only the 15 least significant bits used.
+func (rr *OPT) Z() uint16 {
+	return uint16(rr.Hdr.Ttl & 0x007FFF)
```

Comment:
Nit: write the mask value the same way in both functions. Either leading zeroes or no leading series.